src/NzbDrone.Core/Download/Clients/Hadouken/AuthenticationType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NzbDrone.Core.Download.Clients.Hadouken
+{
+    public enum AuthenticationType
+    {
+        None = 0,
+        Basic,
+        Token
```

Comment:
Reading the release notes 5.1 dropped support for anonymous and token authentication, I think we should clean this up and set a minimum required version to 5.1 (I assume we can check the version in the API and have a test for that when testing the connection).